source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -339,6 +383,794 @@ TEST_F(NiRFmxWLANDriverApiTests, TXPFromExample_FetchData_DataLooksReasonable)
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_data_constellation_trace_response);
+  EXPECT_NE(0.0, ofdm_mod_acc_fetch_data_constellation_trace_response.data_constellation(0).real());
+  EXPECT_NE(0.0, ofdm_mod_acc_fetch_data_constellation_trace_response.data_constellation(0).imaginary());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response);
+  EXPECT_EQ(-26, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.x0());
+  EXPECT_EQ(1, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.dx());
+  EXPECT_EQ(53, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.chain_rms_evm_per_subcarrier_mean_size());
+  EXPECT_EQ(53, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.chain_rms_evm_per_subcarrier_mean().size());
+  EXPECT_NE(0.0, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.chain_rms_evm_per_subcarrier_mean(0));
+}
+
+TEST_F(NiRFmxWLANDriverApiTests, OFDMModAccMIMOFromExample_FetchData_DataLooksReasonable)
+{
+  const auto NUMBER_OF_DEVICES = 2;
+  const auto MAX_SELECTOR_STRING_LENGTH = 256;
+  const auto NUMBER_OF_FREQUENCY_SEGMENTS = 1;
+  const auto NUMBER_OF_RECEIVE_CHAINS = 2;
+  char resourceNames[NUMBER_OF_DEVICES][MAX_SELECTOR_STRING_LENGTH] = { "RFSA1", "RFSA2" };
+  char commaSeparatedResourceName[MAX_SELECTOR_STRING_LENGTH * 2 + 1];
+  GetCommaSeparatedStringFromArray((char*)resourceNames, NUMBER_OF_DEVICES, MAX_SELECTOR_STRING_LENGTH, commaSeparatedResourceName);
+  char selectedPorts[NUMBER_OF_DEVICES][MAX_SELECTOR_STRING_LENGTH] = { "", "" };
+  std::vector<float64> centerFrequencyArray { 5.180000e9, 5.260000e9 };
+  char portString[NUMBER_OF_DEVICES][MAX_SELECTOR_STRING_LENGTH];
+  char selectedPortsString[NUMBER_OF_DEVICES][MAX_SELECTOR_STRING_LENGTH];
+  char selectedPortsStringCommaSeparated[MAX_SELECTOR_STRING_LENGTH * 2 + 1];
+  std::vector<float64> referenceLevelArray { 0.0, 0.0 };
+  std::vector<float64> externalAttenuationArray { 0.0, 0.0 };
+  auto instr_stub = create_stub<nirfmxinstr_grpc::NiRFmxInstr>();
+  auto session = init_session(stub(), PXI_5663E, commaSeparatedResourceName);
+  EXPECT_SUCCESS(session, client::cfg_frequency_reference(stub(), session, "", FREQUENCY_REFERENCE_SOURCE_PXI_CLK, 10e6));
+  EXPECT_SUCCESS(session, client::cfg_number_of_frequency_segments_and_receive_chains(stub(), session, "", NUMBER_OF_FREQUENCY_SEGMENTS, NUMBER_OF_RECEIVE_CHAINS));
+  for (int i = 0; i < NUMBER_OF_FREQUENCY_SEGMENTS; ++i)
+  {
+    auto segment_string_response = client::build_segment_string(stub(), "", i);
+    EXPECT_SUCCESS(session, segment_string_response);
+    EXPECT_SUCCESS(session, client::cfg_frequency(stub(), session, segment_string_response.selector_string_out(), centerFrequencyArray[i]));
+  }
+  for (int i = 0; i < NUMBER_OF_DEVICES; ++i)
+  {
+    // TODO: Fix build_port_string parameter invalid issue.
```

Comment:
Need task to track

Reply:
Added Bug AB#1830349 under the WLAN feature to investigate the `build_port_string` issues. Added a reference to that Bug in the three spaces where the `build_port_string` calls are commented out.